contracts/escrow/dapps/CompoundFacet.sol

```diff
@@ -55,17 +56,38 @@ contract CompoundFacet is PausableMods, DappMods {
     ) public paused("", false) onlyBorrower(loanID) {
         ICErc20 cToken = AssetCTokenLib.get(tokenAddress);
 
-        LibEscrow.e(loanID).setTokenAllowance(tokenAddress, address(cToken));
-
-        bytes memory result = LibEscrow.e(loanID).callDapp(
-            address(cToken),
-            abi.encodeWithSelector(ICErc20.mint.selector, amount)
-        );
-
-        require(
-            abi.decode(result, (uint256)) == LibCompound.NO_ERROR,
-            "Teller: compound deposit error"
-        );
+        if (
+            keccak256(abi.encodePacked(cToken.symbol())) ==
+            keccak256(abi.encodePacked("cETH"))
+        ) {
+            // Unwrap WETH to deposit into cETH
+            IWETH(tokenAddress).withdraw(amount);
+
+            bytes memory result = LibEscrow.e(loanID).callDapp(
```

Comment:
need to make the `LoansEscrow_V1.callDapp` function `payable` and then here update the call to `callDapp` with:
```solidity
LibEscrow.e(loanID).callDapp{ value: amount }(
```

Reply:
then in the `callDapp` function update to be:
```solidity
        resData_ = Address.functionCallWithValue(
            dappAddress,
            dappData,
            msg.value,
            "Teller: dapp call failed"
        );
```